contracts/LifCrowdsale.sol

```diff
@@ -191,18 +210,41 @@ contract LifCrowdsale is Ownable, Pausable {
 
     uint256 tokens = weiSent.mul(privatePresaleRate);
 
-    require(totalPresaleWei.add(weiSent) <= maxPresaleWei);
-
     totalPresaleWei.add(weiSent);
 
     token.mint(beneficiary, tokens);
   }
 
   // send ether to the fund collection wallet
-  function forwardFunds() onlyOwner {
-    foundationWallet.transfer(this.balance);
-    // TODO
-    // marketMaker.transfer(marketMakerPercentage * this.balance);
+  function forwardFunds() internal {
+
+    // calculate the max amount of wei for the foundation
+    uint256 foundationBalanceCapWei= foundationBalanceCapUSD.mul(weiPerUSD);
+
+    // if the minimiun cap for the market maker is not reached transfer all funds to foundation
+    // else if the min cap for the market maker is reached, create it and send the remaining funds
+    if (this.balance < foundationBalanceCapWei) {
```

Comment:
as we are writing this `if` logic, why don't we add a condition for the TGE minimum balance being reached or not, with a TODO to return the contributed eth to the contributors?

Reply:
:+1:

Reply:
thinking more about this, I see this is not needed because `forwardFunds` is only called when the minimum balance was reached